infra/bin/app.ts

```diff
@@ -1,75 +1,134 @@
 #!/usr/bin/env node
 
 import * as cdk from "@aws-cdk/core";
-import { RdsStack, AppInfo } from "@rylabs/ry-cdk-tools/lib/rds";
+import { RdsStack } from "@rylabs/ry-cdk-tools/lib/rds";
 import { InstanceType, InstanceSize, InstanceClass } from "@aws-cdk/aws-ec2";
-import { RailsPipelineStack, RailsStack } from "@rylabs/ry-cdk-tools/lib/rails";
+import {
+  RailsPipelineStack,
+  RailsStack,
+  AppInfo,
+  Conventions,
+} from "@rylabs/ry-cdk-tools/lib/rails";
 
 const env = {
-  account: "Enter Your AWS Account ID",
-  region: "us-east-1",
+  account: "AWS_ACCOUNT_ID",
+  region: "us-east-2",
 };
 
 const appInfo: AppInfo = {
   // Name of the app, in lower title case format. Use for naming AWS resources.
-  name: "enterAppName",
+  name: "app name here",
 
   // Short name of the environment. Use for naming AWS resources.
-  environment: "envShortName",
+  environment: "staging",
 
   // Environment name.
-  orgName: "Enter Your Organization Name",
+  orgName: "Company name",
 
   // Author name.
-  author: "Enter Your Name",
+  author: "Your name here",
 };
 
 // Github repo owner name
-const ownerName = "enter-repo-owner-name";
+const ownerName = "Github repo owner name";
 
 // Github repo name
-const repoName = "enter-repo-name";
+const repoName = "Github repo name";
+
+// Branch this environment deploys from
+const branchName = "develop";
 
 // AWS VPC ID
-const vpcId = "enter-vpc-id";
+const vpcId = "vpc-ID#####";
+
+// Ruby stack
+const solutionStackName = "64bit Amazon Linux 2 v3.2.2 running Ruby 2.7";
 
-// Server EC2 Instance type
-const instanceType = InstanceType.of(InstanceClass.T2, InstanceSize.MEDIUM);
```

Comment:
I had an update that I forgot to check in where I'm setting the EC2 instances for EB – we should check that in as well.  We should set the instance type to be t3.medium, because we were running out of memory with t3.micro